hr_expense_product_policy/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import test_hr_expense
```

Comment:
Your tests only [miss](https://travis-ci.org/OCA/hr/jobs/142426834#L1535) 6 lines of code.
Not bad!